src/data/account/Account.js

```diff
@@ -44,6 +49,7 @@ export const resetPasswordAsync = async (
       method: 'PUT',
       url: endpoints.PASSWORD_RESET,
       headers: {'Content-Type': 'application/json'},
+//TODO: API is expecting a token, but API works fine without passing token probably because the token is 'undefined'
```

Comment:
Although even undefined is acceptable, shall we add a validation on token and throw an exception if it's not in a valid form?

Reply:
yes, ticket: https://carpal.atlassian.net/browse/EB-1388